src/lib/viewers/BaseViewer.js

```diff
@@ -976,7 +976,7 @@ class BaseViewer extends EventEmitter {
         });
 
         // Add a custom listener to scroll to the specified annotation
-        this.addListener('scrolltoannotation', data => this.annotator.scrollToAnnotation(data));
+        this.addListener('scrolltoannotation', this.handleScrollToAnnotation);
```

Comment:
Will we update BUIE to emit this event?

Reply:
Yeah, I am getting the other PR's together.